include/h2o/socketpool.h

```diff
@@ -95,29 +101,15 @@ typedef struct st_h2o_socketpool_t {
 
 typedef struct st_h2o_socketpool_connect_request_t h2o_socketpool_connect_request_t;
 
-typedef void (*h2o_socketpool_connect_cb)(h2o_socket_t *sock, const char *errstr, void *data, h2o_socketpool_target_t *target);
+typedef void (*h2o_socketpool_connect_cb)(h2o_socket_t *sock, const char *errstr, void *data, h2o_url_t *url);
 /**
- * initializes a socket loop
+ * initializes a static socket pool
```

Comment:
How about using words: _specific_ and _global_ instead of (_static_ and _dynamic_)?